server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -472,6 +469,8 @@ public class ConfigurationManagerImpl extends ManagerBase implements Configurati
     public static final ConfigKey<Boolean> ENABLE_DOMAIN_SETTINGS_FOR_CHILD_DOMAIN = new ConfigKey<Boolean>(Boolean.class, "enable.domain.settings.for.child.domain", "Advanced", "false",
             "Indicates whether the settings of parent domain should be applied for child domain. If true, the child domain will get value from parent domain if its not configured in child domain else global value is taken.",
             true, ConfigKey.Scope.Global, null);
+    public static final ConfigKey<Integer> HOST_RESERVED_MEM_MB = new ConfigKey<Integer>("Advanced", Integer.class, HOST_RESERVED_MEM_MB_STRING, "1024",
```

Comment:
```suggestion
    public static final ConfigKey<Integer> HOST_RESERVED_MEM_MB = new ConfigKey<>("Advanced", Integer.class, HOST_RESERVED_MEM_MB_STRING, "1024",
```

Reply:
Good catch. Btw, the other the configs have type in angle brackets, should I remove those, as well? Maybe in separate pull request?

Reply:
can do in this one or a separate one. doesn´t matter.

Reply:
Cool, will do that in separate one. In case someone want to back port it, he/she doesn't have to get unnecessary code changes.